kubernetes/cache/istio.go

```diff
@@ -20,8 +20,7 @@ type (
 
 func (c *kialiCacheImpl) CheckIstioResource(resource string) bool {
 	// This list can be populated from configuration in the future
-	istioResources := []string{"VirtualService", "DestinationRule", "Gateway", "ServiceEntry"}
-	for _, r := range istioResources {
+	for _, r := range c.cacheIstioTypes {
```

Comment:
Considering this method is called so often (every lookup), it makes sense to use a map here instead of always doing a linear search.

Reply:
+1